.github/workflows/keg-build.yml

```diff
@@ -0,0 +1,43 @@
+
+name: keg-build
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  tests:
+    name: 'Build Repos'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: "Get Changed Files"
+        id: get-changed-files
+        run: bash scripts/ci/getChangedFiles.sh
+
+      - name: "Get Changed Repos"
+        id: get-changed-repos
+        run: bash scripts/ci/getChangedRepos.sh
+
+      - name: "Install Yarn Dependencies"
+        id: install-yarn-dependencies
+        run: bash scripts/ci/runYarnCmd.sh "install"
+
+      - name: "Run Repo Tests"
+        id: run-repo-tests
+        run: bash scripts/ci/runYarnCmd.sh "test"
+
+      - name: "Run Repo Build"
+        id: run-repo-build
+        run: bash scripts/ci/runYarnCmd.sh "build"
+
+      - name: "Commit Repo Changes"
```

Comment:
when this workflow executes. `master` will be 1 ahead from `develop` which might cause conflict on the next push from develop > master

just thinking out loud here; maybe we can:
- add another step to merge down master to develop after the commit?
- update the build step to be on develop instead of master. master can be just for publishing on push?

Reply:
I like it, let me look into it

Reply:
Ok, update keg-build action to run on pushes to develop